samples/notebooks/dotnet/05-using-the-planner.ipynb

```diff
@@ -118,295 +118,410 @@
    },
    "outputs": [
     {
-     "name": "stdout",
```

Comment:
I should probably revert this file and only make changes after nuget packages are spun.